stdlib/task/src/main/ballerina/src/task/listener.bal

```diff
@@ -46,7 +46,7 @@ public type Listener object {
     #
     # + s - Service to attach to the listener
     # + name - Name of the service
-    # + return - Returns nil if attaching the service is successful
+    # + return - Nil if attaching the service is successful
```

Comment:
```suggestion
    # + return - Nil if attaching the service is successful
```
@daneshk - Here, Nil should be (), isn't it?

Reply:
Fixed